cime_config/machines/config_machines.xml

```diff
@@ -1145,7 +1145,6 @@
       <executable>srun</executable>
       <arguments>
         <arg name="num_tasks">--mpi=pmi2 -l -n {{ total_tasks }} -N {{ num_nodes }} --kill-on-bad-exit </arg>
-        <arg name="binding">--cpu_bind=cores</arg>
```

Comment:
why is this removed?

Reply:
It is replaced by identical SLURM_CPU_BIND env-var, just to keep SMT available for MPI-only runs too.